reflex/state.py

```diff
@@ -1110,3 +1127,104 @@ def _convert_mutable_datatypes(
         )
 
     return field_value
+
+
+class ClientStorageBase:
```

Comment:
Should these inherit from `rx.Base` to get Pydantic support? Most of our other classes inherit from there